tests/cephadm/test_cephadm_upgrade.py

```diff
@@ -35,7 +113,27 @@ def run(ceph_cluster, **kwargs) -> int:
     LOG.info("Upgrade Ceph cluster...")
     config = kwargs["config"]
     orch = Orch(cluster=ceph_cluster, **config)
+
+    upgrade_completed = False
+
+    def indefinite_run(pool="", seconds=10):
+        """
+        run indefinite loop of rados bench IOs with pool provided
+            - write, seq and cleanup
+
+        """
+        rados = RadosBench(ceph_cluster.get_nodes(role="client")[0], pool)
+        while not upgrade_completed:
+            try:
+                rados.seq(seconds)
+            finally:
+                rados.cleanup()
+        rados.delete_pool()
+        return "Done....."
```

Comment:
nice template though, we can add this too outside this file, probably make a separate package